editor/src/components/canvas/dom-walker.ts

```diff
@@ -135,6 +137,50 @@ function lazyValue<T>(getter: () => T) {
   }
 }
 
+function getAttributesComingFromStyleSheets(element: HTMLElement): Set<string> {
+  let appliedAttributes = new Set<string>()
+  const sheets = document.styleSheets
+  try {
+    for (var i in sheets) {
+      var rules = sheets[i].rules || sheets[i].cssRules
```

Comment:
Can we not just `for...of` this instead if we don't care about the index otherwise? As it's indexing here twice.

Reply:
I _think_ I tried that in codesandbox and it crashed because IIRC `sheets` is a thing that has keys but is not iterable or something horrifying like that

Reply:
Ooooh... so maybe `fastForEach` wouldn't work here then :/